packages/scene-composer/src/hooks/useCallbackWhenNotPanning.ts

```diff
@@ -16,7 +16,7 @@ const useCallbackWhenNotPanning = (callback, deps, acceptableDriftDistance = 1)
   const lastPointerDownLocation = useRef<[number, number] | null>(null);
 
   const onPointerUp = useCallback(
-    (e: ThreeEvent<MouseEvent>) => {
+    (e: ThreeEvent<MouseEvent> | PointerEvent) => {
```

Comment:
Not sure if it is necessary right now but should we use `ThreeEvent<PointerEvent>` vs `ThreeEvent<MouseEvent>`? Not your change just thinking out loud